code/datums/perks/racial.dm

```diff
@@ -512,23 +512,26 @@
 
 /datum/perk/folken_healing
 	name = "Folken Photo-Healing"
-	desc = "As a Folken, you can use the light to heal wounds."
+	desc = "As a Folken, you can use the light to heal wounds, standing in areas of bright light will increase your natural regeneration."
 	passivePerk = TRUE
 
 /datum/perk/folken_healing/young
+	name = "Folken Photo-Healing"
+	desc = "As a Folken, you can use the light to heal wounds, standing in areas of bright light will increase your natural regeneration. Due to your comparitively young age, you heal much faster than older folken."
 
 
 
 ////////////////////////////// Mycus Perks
 
 /datum/perk/dark_heal
 	name = "Mycus Regeneration"
-	desc = "As a mycus, you heal as long as you are in the darkness."
+	desc = "As a mycus, you heal as long as you are in the darkness, increasing your natural regeneration."
 	passivePerk = TRUE
 
 /datum/perk/mushroom_follower
-	name = "Spawn Mushroom Follower"
-	desc = "Mushroom followers produces random healing chems when fed enough food."
+	name = "Spawn Shroomling"
+	desc = "Shroomlings are animal-intelligence mycus capable of following simple orders like 'Shroomling 'Name' Follow.' and 'Shroomling 'Name' Stop.' who will stay by you when ordered. While capable of fighting, they are quite weak, the \
```

Comment:
I think it should mention that you need to mention their name & follow/stop in the same sentence.

Reply:
'Shroomling 'Name' Follow.' is in the perk name.

Reply:
I meant that, from the way it is written, it seems at if it **must** follow that format, when you just need to mention their names (which include the 'Shroomling' in front, I don't know why you did that) and 'follow' or 'stop' in the same sentence.